requirements.txt

```diff
@@ -1,2 +1,2 @@
-jinja2==3.1.2
-hm_pyhelper==0.13.59
+jinja2==3.0.2
```

Comment:
why are we downgrading jinja?

Reply:
:) As I've explained, it's due to some dark fate. Reverted back.